python/ray/autoscaler/kubernetes/kubectl-rsync.sh

```diff
@@ -18,7 +18,8 @@ shift
 if [ "X$pod" = "X-l" ]; then
     pod=$1
     shift
-    namespace="-n $1"
+    # Space before $1 leads to namespace errors
+    namespace="-n$1"
```

Comment:
Where does this come up? Seems strange - maybe a quoting issue somewhere?

Reply:
I checked this locally and in a pod (running on latest autoscaler build). Actually it's quite easy to check:

```
> kubectl "-n ray" get pods               
No resources found in  ray namespace.
```
In contrast:
```
> kubectl "-nray" get pods 
NAME                          READY   STATUS    RESTARTS   AGE
ray-head-7dc9854874-jsflq     1/1     Running   0          23h
```
Note that this doesn't depend on the quotation marks:
```
> kubectl -nray get pods 
NAME                          READY   STATUS    RESTARTS   AGE
ray-head-7dc9854874-jsflq     1/1     Running   0          23h
```

This is my client version:
```
> kubectl version --client
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6", GitCommit:"dff82dc0de47299ab66c83c626e08b245ab19037", GitTreeState:"clean", BuildDate:"2020-07-15T16:58:53Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"darwin/amd64"}
```

Could you maybe check this on your machine?

Reply:
E.g. in mysql cli it's also common to have the argument value directly after the flag, e.g. `mysql -pPASSWORD`

Reply:
I see, had the same problem! Thanks for the clarification.